generate/beat/{{cookiecutter.beat}}/main.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 func main() {
-	err := beat.Run("{{cookiecutter.beat}}", "", beater.New)
+	err := beat.Run("{{cookiecutter.beat}}", {{cookiecutter.beat}}Version, beater.New)
```

Comment:
We left it empty on purpose, so the default version is the same version as is used in libbeat. The main problem currently is that every time cookiecutter is used based on the tutorial, master is used and we should for example have in the docs pointing it to 5.1.